src/app/@ansyn/ansyn/components/ansyn-footer/ansyn-footer.component.less

```diff
@@ -23,10 +23,6 @@
 				left: 50%;
 				transform: translateX(-50%); // center element in  parent
 			}
-
```

Comment:
Please un-delete this.
Then move the "top" property that you added in the rtl section, up to here.